### ISSUE TITLE
Duplicate mount point with Docker

### ISSUE THREAD

**Issue by Z-Zen** [nextflow-io/nextflow#1863]
### Expected behavior and actual behavior

When using `-v /path/:/path/` in `nextflow.config`, it gives an error about duplicate mount point for the first process. This looks like a duplicate of [#384](https://github.com/nextflow-io/nextflow/issues/384). Normally, this should not happen if it's specified only once in the `nextflow.config`.

### Steps to reproduce the problem
```nextflow
profiles {
	docker {
		docker {
			enabled = true
			runOptions = "-v /dev/shm/:/dev/shm/ -v /media/:/media/ --memory-swap 0 --cap-add SYS_NICE"
			fixOwnership = true
		}
	}
}

process {
	executor = 'slurm'
	container = 'imageTest'
	clusterOptions = "-w node1"
	
	withName: process1 {
		memory = 1.GB
		cpus = 1
		cache = 'lenient'
	}
	withName: process2 {
		memory = 1.GB
		cpus = 1
		cache = 'lenient'
	}
...
	withName: processX {
		memory = X.GB
		cpus = Y
		cache = 'lenient'
	}
}
workDir = '/media/tmp/'  // here the "path" is the same as the mount point for docker above.
```

### Program output 

```
docker: Error response from daemon: Duplicate mount point: /media.
See 'docker run --help'.
```

### Environment 

* Nextflow version: `v20.10.0, build 5430 (01-11-2020 15:14 UTC)`
* Java version: `openjdk version "1.8.0_275"`
* Operating system: Debian 9 (stretch), Docker in Debian 9 too
* Bash version: `GNU bash, version 4.4.12`

### Additional context

I tried to delete the mount point `-v /media/:/media/`  and I got the error:
```bash
touch: cannot touch '.command.trace': Permission denied
```

Also, in the `.command.run`, I see the duplicated option:
```bash
nxf_launch() {
    docker run -i --memory 1024m -e "NXF_DEBUG=${NXF_DEBUG:=0}" \
-e "NXF_OWNER=$(id -u):$(id -g)" -v /media:/media -v "$PWD":"$PWD" \
-w "$PWD" --entrypoint /bin/bash -v /dev/shm/:/dev/shm/ \
-v /media/:/media/ --cap-add SYS_NICE --memory-swap 0 \
--name $NXF_BOXID imageTest -c "eval $(nxf_container_env); /bin/bash /media/Run/work3/72/07fc096f891586d50ef965ea863990/.command.run nxf_trace"
}
```

**Comment by qscacheri** [comment]
Bump

**Comment by pmoradi2022** [comment]
@Z-Zen were you able to find a fix for this issue?  I have the same problem with my workflow currently.

**Comment by qscacheri** [comment]
@pmoradi2022 my issue was that it was automatically mounting my home home directory without me realizing and then I tried to mount it again manually

**Comment by pmoradi2022** [comment]
@qscacheri How do I prevent nextflow from automatically mounting my home dir?
